### DISCUSSION TITLE
starstruck' bug

### DISCUSSION THREAD

**Discussion by AllenZYJ** [community/community#21207] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/34624932/180677894-1af6942c-9028-4a61-abb8-9df8e33e2159.png)
starstruck's stars num is wrong , it seem fork's num
![image](https://user-images.githubusercontent.com/34624932/180677988-ecff0ca8-3bd0-4632-b951-34095b2f0665.png)

**Comment by radar-blip** [comment]
Its not wrong, the Starstruck achievement also means a repository with 16 stars. So you must have a repo with 16 stars.
Your Edge-Computing-Engine has 90+ stars but your achievement badge will only show 16 until you reach the next tier (level) which is 128, then you will get credit for 128 stars.

Here is more info on achievements : https://github.com/drknzz/GitHub-Achievements
